t/146-tcp-socket-receive-bsd-style.t

```diff
@@ -0,0 +1,94 @@
+# vim:set ft= ts=4 sw=4 et fdm=marker:
+
+use Test::Nginx::Socket::Lua 'no_plan';
```

Comment:
Better to avoid using `no_plan` in the lua-nginx-module test suite.
What need to do is just copy the run test number from `Bad plan` error message.